src/main/java/net/coderbot/iris/mixin/MixinTranslationStorage.java

```diff
@@ -0,0 +1,53 @@
+package net.coderbot.iris.mixin;
+
+import net.coderbot.iris.Iris;
+import net.minecraft.client.resource.language.LanguageDefinition;
+import net.minecraft.client.resource.language.TranslationStorage;
+import net.minecraft.resource.ResourceManager;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Mixin(TranslationStorage.class)
+public class MixinTranslationStorage {
+
+    //this is needed to keep track of which language code we need to grab our lang files from
+    private static List<String> languageCodes = new ArrayList<>();
+
+    private static final String LOAD = "load(Lnet/minecraft/resource/ResourceManager;Ljava/util/List;)Lnet/minecraft/client/resource/language/TranslationStorage;";
+
+    @Shadow @Final private Map<String, String> translations;
```

Comment:
I prefer the following style:
```suggestion
    @Shadow
    @Final
    private Map<String, String> translations;
```